pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -200,12 +207,64 @@ public String getQueryConsoleWebappPath() {
     return ControllerConf.class.getClassLoader().getResource("webapp").toExternalForm();
   }
 
-  public void setQueryConsoleUseHttps(boolean useHttps) {
-    setProperty(CONSOLE_WEBAPP_USE_HTTPS, useHttps);
+  public void setUseHttp(boolean useHTTP) {
+    setProperty(USE_HTTP, useHTTP);
   }
 
-  public boolean getQueryConsoleUseHttps() {
-    return containsKey(CONSOLE_WEBAPP_USE_HTTPS) && getBoolean(CONSOLE_WEBAPP_USE_HTTPS);
+  public boolean getUseHttp() {
+    if (!containsKey(USE_HTTP)) {
+      return true;
+    } else {
+      return (boolean) getBoolean(USE_HTTP);
+    }
+  }
+
+  public void setUseHttps(boolean useHttps) {
+	setProperty(USE_HTTPS, useHttps);
+  }
+  
+  public boolean getUseHttps() {
+	return containsKey(USE_HTTPS) && getBoolean(USE_HTTPS);
+  }
+  
+  public void setKeyStoreFile(String keyStoreFile) {
+	setProperty(KEYSTORE_FILE, keyStoreFile);
+  }
+
+  public String getBrokerLoadbalancerAddress() {
```

Comment:
There can be multiple brokers in the cluster, each serving some subset of the tables. Not sure what the admin is supposed to set for the broker load balancer here